@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/damage/fn_damageM230.sqf

```diff
@@ -86,7 +86,7 @@ if (_heatfactor > 1200) then {
 _rand = random _heatfactor;
 
 if (_rand > _heatchance) then {
-    //["fza_ah64_bt_gun", 0.5, "fza_ah64_bt_jammed", 1] spawn fza_fnc_playAudio;
+    //Mastercaution here
```

Comment:
When the gun fails there is no alert to the crew via audio. The WPN page will display FAIL on the gun and the gun will be colored yellow.